### DISCUSSION TITLE
part-ii/2019-119/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#117] [Question comments]
Part II 2019 Probability and Measure: Paper 4, Section II, K 

https://questions.tripos.org/part-ii/2019-119/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

a) (ii) implies (i): Fatou.

</details>

**Comment by eha1234** [comment]
<details><summary>More spoilers than the comment above for (a) </summary>

(i) => (ii) Note $\mathbb{E}(X_n^2) - \mathbb{E}(X^2) = \mathbb{E}(X_n - X)^2+ 2\mathbb{E}X(X_n-X)$. Now apply Holder's inequality to $\mathbb{E}X(X_n-X)$

(ii) => (i) Not sure if this is the best way to do it but... Have $(X_n -X)^2 \leq 4(X_n^2 + X^2)$. Apply Fatou to $4(X_n^2+X^2) - (X_n-X)^2$, aiming for a bound on $\limsup (X_n-X)^2$
</details>
